frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -405,16 +294,67 @@ const MultiStepWizard = (): JSX.Element => {
   /**
    * State to manage Parameter `Data`
    */
-  const [parameter, setParameter] = React.useState<Data>();
+  const [parameter, setParameter] = React.useState<InternalState>();
   /**
    * State to manage all Parameter `Data`
    */
-  const [multiParams, setMultiParams] = React.useState<Data[]>([]);
+  const [multiParams, setMultiParams] = React.useState<InternalState[]>([]);
   /**
    * State to manage if button submitted is disabled
    */
   const [isSubmitDisabled, setIsSubmitDisabled] = React.useState(false);
 
+  const alarmValuesSpO2 = useSelector(getSpO2AlarmLimitsRequest, shallowEqual);
+  const alarmValuesHR = useSelector(getHRAlarmLimitsRequest, shallowEqual);
+  const parametersFiO2 = useSelector(getParametersFiO2);
+  const parametersFlow = useSelector(getParametersFlow);
+
+  const determineInput = useCallback(
+    (state: string) => {
+      const stateMap = new Map<string, InternalState>([
+        ['spo2', createInternalState('SpO2', 'spo2', PERCENT, false, true, alarmValuesSpO2, -1)],
+        [
+          'hr',
+          createInternalState('HR', 'hr', BPM, false, true, alarmValuesHR, -1, null, null, 0, 200),
+        ],
+        [
+          'fio2',
+          createInternalState(
+            'FiO2',
+            'fio2',
+            PERCENT,
+            true,
+            false,
+            { lower: 0, upper: 0 },
```

Comment:
Would it be more appropriate to have `null` be the initial/default value here, instead of `{ lower: 0, upper: 0 }` which could be an actual combination of alarm limits values specifiable by the user?